src/plugins/notifications.py

```diff
@@ -15,6 +15,11 @@
 @plugin.listener(hikari.StartedEvent, bind=True)
 async def start_all_tasks(plug: utils.Plugin, _: hikari.StartedEvent) -> None:
     async def cron_task(item: config.ConfigNotifications) -> None:
+        probability = t.cast(float, getattr(item, "probability", None))
```

Comment:
Why is this cast needed?

Reply:
yeah, `toml` isn't that advanced. that `item` object is actually an instance of `Config`, not `ConfigNotifications`, so I see no easy way to make `probability` default to `None`...

Reply:
we can also do:
```suggestion
        probability = item.probability if 'probability' in item else None
```
this infers the type correctly without a cast

Reply:
After doing some testing, I can see where the issue lies, but it would be too cumbersome to fix. I like this solution better, it's more readable.